### PR TITLE
Enable mdoc for Scala.js

### PR DESCRIPTION
**Pull request by sideeffffect**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -433,7 +433,7 @@ lazy val benchmarks = project.module
         "org.scala-lang"             % "scala-reflect"  % scalaVersion.value,
         "org.typelevel"             %% "cats-effect"    % "2.2.0",
         "org.scalacheck"            %% "scalacheck"     % "1.15.1",
-        "qa.hedgehog"               %% "hedgehog-core"  % "0.6.0",
+        "qa.hedgehog"               %% "hedgehog-core"  % "0.5.1",
```
Comment (sideeffffect) [inline build.sbt:436]: `0.6.0` version of `hedgehog-core` is nowhere to be found... It seems like it's been yanked from Maven Central

**Comment by sideeffffect** [comment]
Those two red builds need restarting, otherwise this should be good to go
/cc @adamgfraser :pray:

**Comment by mijicd** [comment]
@sideeffffect Restarted.

**Comment by sideeffffect** [comment]
All tests green :tada: ready to be merged

**Pull request merged**